src/plugins/lanplus/lanplus.c

```diff
@@ -1735,6 +1736,26 @@ ipmi_lanplus_build_v2x_msg(
 								msg + IPMI_LANPLUS_OFFSET_PAYLOAD, /* output */
 								&(payload->payload_length));       /* output */
 
+		if (old_payload_length != payload->payload_length)
+		{
+			len =
+				sizeof(rmcp)                +  // RMCP Header (4)
+				10                          +  // IPMI Session Header
+				2                           +  // Message length
```

Comment:
Aren't there any constants/macros already defined for those values?
I like how you commented the magic numbers, but using consts/macros/sizeof() would be even better.

Reply:
I didn't find constants and macros for "IPMI Session Header" and "Message length" but I found `IPMI_LANPLUS_OFFSET_PAYLOAD` is used in the surrounding code as the sum of these two and the size of RMCP header.

For "pad length" and "next header", I couldn't find the macros and constants so that I added macros for them and used them in the PR.

PTAL. Thanks! diff: https://github.com/ipmitool/ipmitool/compare/89fd59107c63fecea0cfa7b754714075db01083c..e9d837f810d79c6fed142bcf833e3b511052b302